src/stochprog.jl

```diff
@@ -5,20 +5,25 @@ using LightGraphs
 """
     AbstractStochasticProgram <: LightGraphs.AbstractGraph{Int}
 
-Stochastic program instance
+Stochastic program instance.
 """
 abstract type AbstractStochasticProgram <: LightGraphs.AbstractGraph{Int} end
 
 """
     stochasticprogram(args...)
 
-Creates a stochastic program from the arguments
+Create a stochastic program from the arguments.
 """
 function stochasticprogram end
 
 ## Node
 
-# The type of nodes is `Int` for compatibility with LightGraphs' nodes
+"""
+    AbstractNode
+
+Node instance.
+"""
+abstract type AbstractNode end
```

Comment:
I introduced this new object because I would like to have a node object inside StochOptInterface. 
But maybe you would prefer to have a `AbstractNodeData` instead.

Reply:
Node `AbstractNode` is a good idea